src/components/bcros/filing/common/FiledLabel.vue

```diff
@@ -14,7 +14,7 @@
           <BcrosTooltipDate :date="filing.submittedDate" />)
         </span>
         <UDivider orientation="vertical" :ui="{ border: { base: 'border-gray-600'} }" />
-        <span>
+        <span v-if="filing.name !== FilingTypes.REGISTRARS_NOTATION">
```

Comment:
Actually, there are other filings that don't show "effective as of ...". For example, a Registrar's Order and Court Order -- ref https://dev.business.bcregistry.gov.bc.ca/BC0883763.

Also, I've generally found that it's safer to specify which conditions to show something for, instead of specify which conditions to not show (in case, say, another type is added).

Reply:
This was a change specifically to fix 24443 which only calls out registrars notation. That said I can revise

Reply:
Your fix looks good. I'm not going to bother testing it.

Please resolve this conversation (as I cannot). Thanks.

Reply:
I tested anyway and I see this:

![image](https://github.com/user-attachments/assets/8b4a5ddb-3f52-412f-9c3a-f70fc1b609ae)

Ref: https://business-dashboard-dev--pr-75-gqzc5b9p.web.app/BC0883569

1. should not have vertical bar when "effective as of" isn't displayed.
2. font size in body is too large

Reply:
The font size is not part of this component and has another ticket, the vertical bar however should be removed now

Reply:
OK, then please resolve this conversation (as I cannot). Thanks.